dev_guide/builds.adoc

```diff
@@ -105,6 +105,39 @@ The following options are specific to the
 link:../architecture/core_concepts/builds_and_image_streams.html#source-build[S2I
 build strategy].
 
+[[force-pull]]
+
+=== Force Pull
+
+By default, if the builder Docker image specified in the  build confirguration is available in OpenShift's local,
+internal registry, that image will be used.  However, to override the local image and refresh it from the Docker image
+repository with a newer version, create a `*buildConfig*` with the "forcePull" flag set to true:
+
+====
+
+[source,json]
+----
+{
+  "strategy": {
+    "type": "Source",
+    "sourceStrategy": {
+      "from": {
+        "kind": "ImageStreamTag",
+        "name": "builder-image:latest" <1>
+      },
+      "forcePull": true <2>
+    }
+  }
+}
+----
+
+<1> The builder image being used, where the local version currently stored in the OpenShift internal registry
+may not be up to date with the version in the Docker image repository.
```

Comment:
it's not the local version in the openshift internal registry, it's the local version on the node.  where we're going to pull it from is the internal registry (if that's where the imagestream points.  we could also pull it from any other registry if there's where the imagestream points)